src/main/java/com/softserve/academy/spaced/repetition/service/DeckCommentService.java

```diff
@@ -3,18 +3,51 @@
 import com.softserve.academy.spaced.repetition.domain.Comment;
 import com.softserve.academy.spaced.repetition.domain.DeckComment;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotAuthorisedUserException;
+
 import java.util.List;
 
+/**
+ * This interface works with the comments to decks.
+ */
 public interface DeckCommentService {
-
+    /**
+     * Add a comment to decks
+     *
+     * @return true if operation is successful
+     */
     DeckComment addCommentForDeck(Long deckId, String commentText, Long parentCommentId)
             throws NotAuthorisedUserException;
 
+    /**
+     * Returns a comment with the given identifier
+     *
+     * @param commentId must not be {@literal null}.
+     */
     DeckComment getCommentById(Long commentId);
 
+    /**
+     * Returns all comments to the deck with the given identifier
+     *
+     * @param deckId must not be {@literal null}.
+     * @return list of comments to the deck with the given identifier
+     */
     List<Comment> getAllCommentsForDeck(Long deckId);
 
+    /**
+     * Updates comment with the given identifier with the new text
+     *
+     * @param commentId   must not be {@literal null}
+     * @param commentText must not be {null}
+     * @return true if operation is successful
+     */
+
     DeckComment updateCommentById(Long commentId, String commentText);
 
+    /**
+     * Delete comments to the deck with the given identifier
```

Comment:
`Delete comments to the deck with the given identifier` 
1. check grammar
2. `Delete comments` it means than this methods may delete few comments? or only one?

Reply:
corrected